modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/OLSLoad_conf.pm

```diff
@@ -33,7 +33,7 @@ package Bio::EnsEMBL::Production::Pipeline::PipeConfig::OLSLoad_conf;
 
 use strict;
 use base ('Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf');
```

Comment:
You could consider using 'Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf' instead. That inherits from the HiveGeneric_conf, but adds in some useful stuff, to say redefining things. In this case, there are 'user' and 'email' parameters already defined, so you can just use `$self->o('user')` and `$self->o('email')` in the payloads; and also an 'ensembl_release' parameter, which defaults to the version of your local ensembl repo, that could replace 'ens_version'.